src/Cli/func/StaticResources/requirements.txt

```diff
@@ -1,5 +1,5 @@
-﻿# Do not include azure-functions-worker in this file

```

Comment:
Is this warning no longer relevant?

Reply:
This was an old warning log. It's not relevant since it only works for python version 3.9 and below which are deprecated. No users are actively using it.